### PR TITLE
Add Tab component

### PR DESCRIPTION
**Pull request by krishnakant-technossus**
**Note:**
**- Slated for v0.0.34**
**- This PR is derived from PR** #95 

Tab component.

Here's how it looks:
Screenshot_20220516-152157_Mau

### CODE REVIEW

**Review by ajaybushel — APPROVED** [review]

**Comment by ajaybushel** [comment]
Only suggestion is, do we need an inverse variant of this component?

**Review by ajaybushel — CHANGES_REQUESTED** [review]
I don't think we should be calling this component UpperTabs. Please rename this to "Tabs".

**Comment by krishnakant-technossus** [comment]
> Only suggestion is, do we need an inverse variant of this component?

Well I don't think because I checked in sketch design and there is no tabs shown over the dark background. only I can see the tabs over the light background. But if you say and it may require in future then we can add inverse mode too.

**Comment by krishnakant-technossus** [comment]
> I don't think we should be calling this component UpperTabs. Please rename this to "Tabs".

Okay I will do this.

**Review by stevekardian — APPROVED** [review]

**Review by stevekardian — APPROVED** [review]

File: App.tsx [hunk 1]
```diff
@@ -275,6 +277,21 @@ export const App: React.FC<IProps> = (): React.ReactElement => {
...
+    {
+      label: 'Tab2',
+      value: 1,
+    },
+    {
+      label: 'Tab3',
+      value: 2,
+    },
+  ];
+
```
Comment (raoufr) [inline App.tsx:294]: I know this was not introduced by this PR, but the url in line #296 seems wrong to me. 
Any ideas @ajaybushel @mdjacobs @bittu1028?
I would think we do not want to hard code an external URL and if we have to have this probably refer to a local image.

File: src/modules/tabs/tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+// Modules
+import * as React from 'react';
+import {FlatList} from 'react-native';
+import Styled from 'styled-components/native';
+
+// Types
+import {IProps, IStyledText, IStyledTouchable, ITab} from './tabs.types';
```
Comment (raoufr) [inline src/modules/tabs/tabs.tsx:7]: @ajaybushel I think we should follow this sub extension for interfaces as well. But let's discuss and sync up on the convention that should be followed across the board.

**Review by raoufr — APPROVED** [review]
Looking good to me. A couple of unblocking comments for discussion and future follow up PR potentially.

**Comment by raoufr** [comment]
@krishnakant-technossus please address @ajaybushel 's request(s) for changes and also resolve the conflict above before we can merge. Thank you.

**Comment by raoufr** [comment]
@krishnakant-technossus  also please add a brief description in the change log to what has been added with this PR. thanks

**Pull request merged**